broker/src/main/java/io/camunda/zeebe/broker/partitioning/NoOpPartitionAdminAccess.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.partitioning;
+
+import io.camunda.zeebe.broker.Loggers;
+import io.camunda.zeebe.util.sched.future.ActorFuture;
+import io.camunda.zeebe.util.sched.future.CompletableActorFuture;
+import org.slf4j.Logger;
+
+public final class NoOpPartitionAdminAccess implements PartitionAdminAccess {
+
+  private static final Logger LOG = Loggers.SYSTEM_LOGGER;
+
+  @Override
+  public ActorFuture<Void> takeSnapshot() {
+    logCall();
+    return CompletableActorFuture.completed(null);
+  }
+
+  @Override
+  public ActorFuture<Void> pauseExporting() {
+    logCall();
+    return CompletableActorFuture.completed(null);
+  }
+
+  @Override
+  public ActorFuture<Void> resumeExporting() {
+    logCall();
+    return CompletableActorFuture.completed(null);
+  }
+
+  @Override
+  public ActorFuture<Void> pauseProcessing() {
+    logCall();
+    return CompletableActorFuture.completed(null);
+  }
+
+  @Override
+  public ActorFuture<Void> resumeProcessing() {
+    logCall();
+    return CompletableActorFuture.completed(null);
+  }
+
+  private void logCall() {
+    LOG.warn("Received call on NoOp implementation of PartitionAdminAccess");
```

Comment:
:question: Why warning? Is this because we expect this to never happen, or only at the beginning?

Reply:
The way the class is used now, it would only happen during startup. But I don't know where this class will be used in the future. So the log message is to indicate that it's probably a mistake when this is called.

However, happy to reduce log level or remove logging altogether. Please advise.

Reply:
Warning level indicates to users that something may be wrong, but can resolve itself without any intervention, and would only require manual intervention if it persists. If we never expected it to happen for example, then a warning is too low, and if we expected it to happen at sporadic intervals, then it would be too high.

If I understand correctly, we expect it to happen at the moment, but only at startup? So it's effectively indicating that something is not working, but we expect it to recover itself. Warning is fine then.